packages/core/admin/admin/src/content-manager/components/RepeatableComponent/components/Accordion.js

```diff
@@ -13,27 +13,30 @@ export const Footer = styled(Box)`
 `;
 
 export const Content = styled(Box)`
-  border: 1px solid ${({ theme }) => theme.colors.neutral200};
   border-bottom: none;
-  border-radius: ${({ theme }) => theme.borderRadius} ${({ theme }) => theme.borderRadius} 0 0;
   overflow: hidden;
 
+  /* add the borders and make sure the top is transparent to avoid jumping with the hover effect  */
+  & > div > div {
+    border: 1px solid ${({ theme }) => theme.colors.neutral200};
+    border-top-color: transparent;
+  }
+
+  /* the top accordion _does_ need a border though */
+  & > div:first-child > div {
+    border-top: 1px solid ${({ theme }) => theme.colors.neutral200};
+  }
```

Comment:
yay fixed - just another small UI bug appeared 👀 (blue bottom border when accordion active/open but not focused)

<img width="748" alt="image" src="https://user-images.githubusercontent.com/71838159/204018988-55bc234f-43ce-4eb9-a7d6-385c3ffad93b.png">

Reply:
Thanks! I should have just done `border` instead of `border-bottom` fixed now.